pictures/migrations.py

```diff
@@ -0,0 +1,33 @@
+from django.db.migrations import AlterField
+
+from pictures.models import PictureField
+
+
+class AlterPictureField(AlterField):
+
+    reduces_to_sql = False
+
+    reversible = True
+
+    def database_forwards(self, app_label, schema_editor, from_state, to_state):
+        super().database_forwards(app_label, schema_editor, from_state, to_state)
+        from_model = from_state.apps.get_model(app_label, self.model_name)
+        to_model = to_state.apps.get_model(app_label, self.model_name)
+        from_field = from_model._meta.get_field(self.name)
+        to_field = to_model._meta.get_field(self.name)
+
+        if not isinstance(from_field, PictureField):
+            for obj in to_model._default_manager.all().iterator():
+                field_file = getattr(obj, self.name)
+                field_file.save_all()
+            return
+
+        if not isinstance(to_field, PictureField):
+            for obj in from_model._default_manager.all().iterator():
+                field_file = getattr(obj, self.name)
+                field_file.delete_all()
+            return
+
+        for obj in to_model._default_manager.all().iterator():
+            field_file = getattr(obj, self.name)
+            field_file.update_all(from_aspect_ratios=from_field.aspect_ratios)
```

Comment:
This might fail if one chooses to switch from a Filed which does not have any aspect_ratios.

Reply:
hm… good point, I'll add a test for that